src/utils/cache.rs

```diff
@@ -120,7 +125,7 @@ impl CachedWithCode {
     }
 
     pub(super) fn is_error(&self) -> bool {
-        500 > self.code && self.code >= 400
+        200 < self.code && self.code >= 400
```

Comment:
So if `self.code` is greater than 200 then it is an error?  That would mean 201 is an error.

Reply:
should be `self.code < 200 || self.code >= 400`

Reply:
not sure in what mental state I was in when I wrote that "correction"